frontend/src/components/Admin/CoffeeChats/CoffeeChats.tsx

```diff
@@ -30,39 +42,99 @@ const CoffeeChats: React.FC = () => {
     });
   }, [isLoading]);
 
+  const handleMemberClick = (member: IdolMember) => {
+    setIsChatLoading(true);
+    setSelectedMember(member);
+
+    CoffeeChatAPI.getCoffeeChatsByUser(member).then((coffeeChats) => {
+      setSpecificApprovedChats(coffeeChats.filter((chat) => chat.status === 'approved'));
+      setSpecificPendingChats(coffeeChats.filter((chat) => chat.status === 'pending'));
+      setSpecificRejectedChats(coffeeChats.filter((chat) => chat.status === 'rejected'));
+      setIsChatLoading(false);
+    });
+
+    CoffeeChatAPI.getCoffeeChatBingoBoard().then((board) => setSpecificBingoBoard(board));
+  };
+
   return (
-    <div>
-      <div className={[styles.formWrapper, styles.wrapper].join(' ')}>
-        <h1>Review Coffee Chats</h1>
-      </div>
-      <div className={styles.wrapper}>
-        <div className={styles.bingo_board}>
-          {bingoBoard.flat().map((category, index) => (
-            <div key={index}>
-              <Link
-                key={category}
-                href={{
-                  pathname: `/admin/coffee-chat-details/${hashString(category)}`,
-                  query: { category }
-                }}
-              >
-                <div className={styles.bingo_cell}>
-                  <div style={{ display: 'flex', flexDirection: 'column' }}>
-                    <div className={styles.bingo_text}>{category}</div>
-                    <div className={styles.pending_text}>
-                      {chatCount.get(category) ? `${chatCount.get(category)} pending` : ''}{' '}
-                    </div>
+    <div className={styles.flexContainer}>
+      <div className="main-content">
+        {selectedMember ? (
+          <div className={[styles.formWrapper, styles.wrapper].join(' ')}>
+            <h2>
+              Bingo Board for {selectedMember.firstName} {selectedMember.lastName}
+            </h2>
+            <CoffeeChatsBingoBoard
+              approvedChats={specificApprovedChats}
+              pendingChats={specificPendingChats}
+              rejectedChats={specificRejectedChats}
+              isChatLoading={isChatLoading}
+              bingoBoard={specificBingoBoard}
+              onCellClick={() => { }}
```

Comment:
We can make `onCellClick` optional by defining the prop as `onCellClick ?: ...` in `CoffeeChatsBingoBoard`. This way we don't need to pass in default values!